### PR TITLE
Version next notifications

### PR DESCRIPTION
**Pull request by Wagnerp**
- General fixes

### CODE REVIEW

File: Source/Krypton Toolkit/Main/Krypton.Toolkit.Suite.Extended.Notifications/UX/Toast/KryptonToastNotificationVersion1.cs [hunk 1]
```diff
@@ -151,6 +151,16 @@ private void InitializeComponent()
             this.ResumeLayout(false);
 
         }
+
+        protected override void Dispose(bool disposing)
```
Comment (Smurf-IV) [inline Source/Krypton Toolkit/Main/Krypton.Toolkit.Suite.Extended.Notifications/UX/Toast/KryptonToastNotificationVersion1.cs:155]: What use is this ?

File: Source/Krypton Toolkit/Main/Krypton.Toolkit.Suite.Extended.Notifications/UX/Toast/KryptonToastNotificationVersion2.cs [hunk 2]
```diff
@@ -165,6 +165,16 @@ private void InitializeComponent()
             this.ResumeLayout(false);
 
         }
+
+        protected override void Dispose(bool disposing)
```
Comment (Smurf-IV) [inline Source/Krypton Toolkit/Main/Krypton.Toolkit.Suite.Extended.Notifications/UX/Toast/KryptonToastNotificationVersion2.cs:169]: What use is this ?

Comment (Wagnerp) [reply Source/Krypton Toolkit/Main/Krypton.Toolkit.Suite.Extended.Notifications/UX/Toast/KryptonToastNotificationVersion2.cs:169]: See my previous comment. Will be redoing the notifications module, as there are a # of bugs (see Discord chat for more info).

File: Source/Krypton Toolkit/Main/Krypton.Toolkit.Suite.Extended.Notifications/UX/Toast/KryptonToastNotificationVersion1.cs (hunk 1 above)
Comment (Smurf-IV) [reply Source/Krypton Toolkit/Main/Krypton.Toolkit.Suite.Extended.Notifications/UX/Toast/KryptonToastNotificationVersion1.cs]: Add the above comment into the code then ;-)

**Pull request merged**